geonode/base/api/tests.py

```diff
@@ -222,23 +312,71 @@ def test_register_users(self):
         self.assertEqual(response.status_code, 201)
         # default contributor group_perm is returned in perms
         self.assertIn('add_resource', response.data['user']['perms'])
+        # Anonymous
+        self.assertIsNone(self.client.logout())
+        response = self.client.post(url, data={'username': 'new_user_1'}, format='json')
+        self.assertEqual(response.status_code, 403)
 
-    def test_delete_users(self):
+    def test_update_user_profile(self):
```

Comment:
To be 100% compatible with the issue requirements is a good option to have here also the test for the group manager